linode/manager
# chore: [M3-4317] - Remove Bluebird.js as a dependency

## Description 📝

- Removes `bluebird` package because it duplicates features built into moden Javascript 🎉

## Major Changes 🔄
- Replaces `Bluebird.reduce` usage with new function called `reduceAsync`, which does the same thing
- Replaced `Bluebird.map` with a combination of `Promise.all` and `map`

## How to test 🧪
- Test creating support tickets in the dev environment with files attached
  - Verify a support ticket gets successfully created when there is attachements
- Test viewing managed issues
  - You can visit `http://localhost:3000/managed/monitors` and turn on the MSW to mock this
  - If you see a Support Ticket linked to an Issue, we can conclude this works
- Test backups drawer
